lib/Resizable.jsx

```diff
@@ -91,16 +92,14 @@ export default class Resizable extends React.Component {
 
   render() {
     let p = this.props;
-    let className = p.className ?
```

Comment:
Good point. Using external CSS seems not the 'proper' react way. I personally find that it is more intuitive and convenient for react developers to tweak the `style` prop rather than include external CSS in the calling code.

How about using inline style but keeping the class name for those who want to tweak the style through external CSS?